test/parallel/test-setproctitle.js

```diff
@@ -1,9 +1,10 @@
 'use strict';
 // Original test written by Jakub Lekstan <kuebzky@gmail.com>
+const common = require('../common');
 
 require('../common');
 // FIXME add sunos support
-if ('linux freebsd darwin'.indexOf(process.platform) === -1) {
+if (!common.isFreeBSD && !common.isOSX && !common.isLinux) {
```

Comment:
This might be more readable as `!(common.isFreeBSD || common.isOSX || common.isLinux)`